packages/orbit-components/src/Box/normalize.js

```diff
@@ -50,52 +43,46 @@ const normalizeSize = (val, property): string => {
   return formatCSS(property, val);
 };
 
-export const normalizeFlex = (key?: "justify" | "align", val: string) => {
-  if ((key === "justify" || key === "align") && val === "end") return "flex-end";
-  if ((key === "justify" || key === "align") && val === "start") return "flex-start";
-  if (key === "justify" && val === "between") return "space-between";
-  if (key === "justify" && val === "around") return "space-around";
-
-  return val;
-};
-
 const normalizeElevation = (val: Elevation, theme: Theme): string => {
   return formatCSS("box-shadow", theme.orbit[`boxShadow${firstToUpper(val)}`]);
 };
 
+const norm = ({ val, key, theme }): string | void => {
+  if (!isDefined(val) || !isDefined(key)) return undefined;
+
+  const all = {
+    padding: normalizeSpacing(val, key, theme),
+    margin: normalizeSpacing(val, key, theme),
+    color: key === "color" && normalizeToken(val, key, "palette", theme),
+    background: key === "background" && normalizeToken(val, key, "palette", theme),
+    height: normalizeSize(val, key),
+    width: normalizeSize(val, key),
+    borderRadius:
+      key === "borderRadius" && normalizeToken(val, "border-radius", "borderRadius", theme),
+    elevation: key === "elevation" && normalizeElevation(val, theme),
+    justify: formatCSS("justify-content", getJustify(val)),
+    align: formatCSS("align-items", getAlign(val)),
+    direction: formatCSS("flex-direction", getDirection(val)),
+    grow: formatCSS("flex-grow", getGrow(val)),
+    shrink: formatCSS("flex-shrink", getShrink(val)),
+    wrap: formatCSS("flex-wrap", getWrap(val)),
+    textAlign: formatCSS("text-align", val),
+    minWidth: formatCSS("min-width", val),
+    maxWidth: formatCSS("max-width", val),
+    maxHeight: formatCSS("max-height", val),
+  };
+
+  if (!all[key]) return formatCSS(key, val);
+
+  return all[key];
+};
+
 const normalize = (object: MediaQueryObject) => ({ theme }: ThemeProps) => {
```

Comment:
Do we have tests for this normalizer? We could easily introduce bugs without them, especially if we perform refactoring 🙂

Reply:
it's covered via normal tests, no need to test normalizer, if output is correct - it means it works fine 🤔